src/components/queue/QueueList/QueueList.tsx

```diff
@@ -24,13 +27,28 @@ export interface QueueListProps {
  */
 const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
     const {currentUser, isTA} = useAuth();
-    const [tickets, ticketsLoading] = useTickets(queue.id, showCompletedTickets);
+    const [tickets, ticketsLoading] = useTickets(queue.id, false);
```

Comment:
Switched the hook to capture all of the tickets and then filter locally -> we need all of the tickets to check whether or not the button should be disabled